src/js/components/CreateServiceModalCatalogPanelOption.js

```diff
@@ -17,7 +17,7 @@ class CreateServicePickerCatalogOption extends React.Component {
       <CreateServiceModalServicePickerOption
         columnClasses={columnClasses}
         onOptionSelect={onOptionSelect.bind(null, {
-          route: "catalog",
+          route: "/catalog",
```

Comment:
As I discussed with @ahoskins this was the only solution found without breaking the other panel buttons but perhaps there is a bug somewhere else.